### ISSUE TITLE
wasm abi remaining stuff

### ISSUE THREAD

**Issue by Centril** [clockworklabs/SpacetimeDB#1790]
- [x] add identity
- [x] remove address from __call_reducer__

**Comment by cloutiertyler** [comment]
I don't believe these actually will be changing their binary representation much. `Address` is going to be renamed `ConnectionId` and passed in as an `Option` to all reducers. To connect and disconnect reducers it will never be `None` (to the user this should not be an option in those two cases). `ConnectionId` will be a u128.

**Comment by bfops** [comment]
@gefjon did the first one; unsure if we want to do the second thing.

May want to do a quick pass on the wasm to make sure we're not missing anything, but we think we're in a good place.
